jetty-core/jetty-server/src/main/java/org/eclipse/jetty/server/handler/CrossOriginHandler.java

```diff
@@ -449,7 +446,7 @@ private boolean isPreflight(Request request)
 
     private boolean isWebSocketUpgrade(Request request)
     {
-        return request.getHeaders().contains(HttpHeader.UPGRADE, "websocket");
+        return request.getHeaders().contains(HttpHeader.SEC_WEBSOCKET_VERSION, "13");
```

Comment:
I think this is fine.

But is there really a point checking if it equals `13`?

I would just do `request.getHeaders().contains(HttpHeader.SEC_WEBSOCKET_VERSION)`.